apps/web/pages/settings/my-account/general.tsx

```diff
@@ -213,6 +215,21 @@ const GeneralView = ({ localeProp, user }: GeneralViewProps) => {
           </>
         )}
       />
+      <div className="mt-8">
+        <Controller
+          name="allowDynamicBooking"
+          control={formMethods.control}
+          render={() => (
+            <SettingsToggle
+              title={t("allow_dynamic_booking")}
```

Comment:
```suggestion
              title={t("dynamic_booking")}
              description{t("allow_dynamic_booking")}
```
Yep, the styling is off. You can do something like this. Please note that `dynamic_booking` doesn't exist in the locale yet, so you'd need to add it to the 'en' locale as well. I believe that should do it. Unless @PeerRich has something to add to this.

Reply:
It would look something like this:
![image](https://github.com/calcom/cal.com/assets/52925846/3b2ea312-838b-4752-8a45-64e8abc2754c)

Reply:
The controls for the Logged-In theme is currently being reviewed, if I'm not mistaken. Currently, it just picks system default theme for the logged-in section. The users will be able to control it better after a PR that is under review, if I'm not mistaken. I mean, the light mode should be fairly similar here. I believe we need the title and description in the toggle and that just adjusts the styling well enough!